src/reducers/planner/planner.ts

```diff
@@ -154,24 +215,26 @@ const planner = (state = initialState, action) => {
         }
         return t;
       });
-
       newPlanners.sort((t1, t2) => t1.arrange_order - t2.arrange_order);
-      const updatedPlanner = newPlanners.find((t) => t.id === state.selectedPlanner.id);
-      return Object.assign({}, state, {
+      const updatedPlanner = newPlanners.find((t) => t.id === state.selectedPlanner?.id);
```

Comment:
위쪽에 if문 추가한다면 여기 optional chaining 빼는 게 어떨까요?

```suggestion
      const updatedPlanner = newPlanners.find((t) => t.id === state.selectedPlanner.id);
```